### ISSUE TITLE
URL bar autocomplete suggests switching to previously closed unloaded tabs

### ISSUE THREAD

**Issue by kn-yami** [JustOff/lull-the-tabs#11]
In both Pale Moon and Basilisk closing an unloaded tab causes URL bar autocomplete feature to attempt switching to that already closed tab if you try to open the same URL again. Here is a video that demonstrates this behaviour: [ltt.mp4.gz](https://github.com/JustOff/lull-the-tabs/files/4741076/ltt.mp4.gz). When this happens nothing appears in the Error Console.
My limited testing suggests that this issue happens regardless of extension and browser version.

**Comment by JustOff** [comment]
Thanks for the detailed report. The irony is that I myself regularly came across this rather annoying issue, but could not find the culprit, which turned out to be my own add-on.

Please check if [lull-the-tabs-1.5.0b1.xpi](https://github.com/JustOff/lull-the-tabs/releases/download/1.4.3/lull-the-tabs-1.5.0b1.xpi) fixes this.

**Comment by kn-yami** [comment]
This beta version fixes this issue for me (I had to quit without saving session storage and then relaunch the browser to make it work, though I was told that it had worked immediately after installation for someone else). I too have experienced this issue for a long time, but I have always blamed it on my rather old browser profile. Only recently, after another Pale Moon user told me that he also had had this problem, I decided to try to find out what causes this issue.
